detekt-rules-libraries/build.gradle.kts

```
@@ -2,8 +2,23 @@ plugins {
     id("module")
 }
 
+val generatedLibrariesConfig: Configuration by configurations.creating {
+    isCanBeConsumed = false
+    isCanBeResolved = true
+}
+
 dependencies {
     compileOnly(projects.detektApi)
     testImplementation(projects.detektTest)
     testImplementation(libs.assertj)
+
+    generatedLibrariesConfig(
+        project(projects.detektGenerator.path, "generatedLibrariesConfig")
+    )
```

I think it would be better to keep them similar to other dependencies (it probably fits easily):
```suggestion
    generatedLibrariesConfig(project(projects.detektGenerator.path, "generatedLibrariesConfig"))
```

---

👍